src/audio/dcblock/tune/sof_dcblock_build_blob.m

```diff
@@ -1,4 +1,4 @@
-function blob8 = dcblock_build_blob(R_coeffs, endian, ipc_ver)
+function blob8 = sof_dcblock_build_blob(R_coeffs, endian, ipc_ver)
```

Comment:
why does the prefix matter if we are already isolated to our own module?

Reply:
Good question, any thoughts @lrgirdwo, should we leave the original file names?